0004-appdata-patch.patch

```diff
@@ -1,20 +1,16 @@
-From c4971714e6def404896200af5bdbfb262ab0ee77 Mon Sep 17 00:00:00 2001
-From: Ryan Gonzalez <rymg19@gmail.com>
-Date: Fri, 9 Jul 2021 12:08:39 -0500
-Subject: [PATCH 5/5] Flatpak-specific appdata changes
```

Comment:
Please leave the original titles! they're a bit more informative.

BTW also I'm not sure how you applied these locally, but I'm guessing you just did `git apply`, and had to rewrite the titles? If you go off of a Git clone of the source repo, you can instead do `git am`, and it'll preserve all of the commit metadata.

Reply:
No, I apply patches with `git am`, the titles are from the commits as `git format-patch` generates them.

Reply:
Do you mean the filename of the patch? In case I changed it to something short instead of the long commit name.

Reply:
Left the original filename intact